majilove/models.py

```diff
@@ -573,7 +585,7 @@ def leader_skill(self):
         _leader_skill_variables['leader_skill_percentage'] = self.final_leader_skill_percentage
         return Photo.LEADER_SKILL_INFO['template'].format(**_leader_skill_variables)
 
-    CROWN_OPTIONS = [150, 200] #Now only 200; change this to a variable
+    CROWN_OPTIONS = [200] #Now only 200; change this to a variable
```

Comment:
Should maybe change this to a variable; tempted to leave it as a list in case other amounts are added in the future.